Documentation/concepts/networking/masquerading.rst

```diff
@@ -136,6 +136,10 @@ The example below shows how to configure the agent via :term:`ConfigMap` and to
     172.16.0.0/12
     192.168.0.0/16
 
+Alternatively, you can pass ``--set ipMasqAgent.config.nonMasqueradeCIDRs='{10.0.0.0/8,172.16.0.0/12,192.168.0.0/16}'``
+and ``--set ipMasqAgent.config.masqLinkLocal=false`` when installing Cilium via Helm to
+configure the via :term: `ip-masq-agent` as above.
```

Comment:
As per the CI docs job failure, I don't think that `ip-masq-agent` is defined as a term currently. You will need to either define the term or just quote the phrase like this:
```suggestion
configure the via ``ip-masq-agent`` as above.
```